apps/app-frontend/src/components/ui/InstanceCreationModal.vue

```diff
@@ -46,18 +46,17 @@
             :show-labels="false"
           />
           <Checkbox
-            v-if="showAdvanced"
             v-model="showSnapshots"
             class="filter-checkbox"
-            label="Include snapshots"
+            label="Snapshots"
```

Comment:
Would make the label "Show all versions" to match the website, since it shows both legacy and snapshot versions